.travis.yml

```diff
@@ -18,6 +18,12 @@ before_script:
     - export SAXON_CP=/tmp/xspec/saxon/saxon9he.jar
     - wget -O ${SAXON_CP} http://central.maven.org/maven2/net/sf/saxon/Saxon-HE/${SAXON_VERSION}/Saxon-HE-${SAXON_VERSION}.jar
     - chmod +x ${SAXON_CP}
+    # install XML Calabash
+    - export XMLCALABASH_VERSION=1.1.15-97
+    - wget -O /tmp/xspec/xmlcalabash-${XMLCALABASH_VERSION}.zip https://github.com/ndw/xmlcalabash1/releases/download/${XMLCALABASH_VERSION}/xmlcalabash-${XMLCALABASH_VERSION}.zip 
+    - sudo apt-get install -y unzip
```

Comment:
Is this `sudo` line inevitable?
[This article](https://docs.travis-ci.com/user/migrating-from-legacy/) seems to recommend not to use `sudo`.